packages/surveycto/src/Adaptor.js

```diff
@@ -26,43 +29,45 @@ export function execute(...operations) {
 }
 
 /**
- * Options provided to the HTTP request
- * @typedef {Object} FormSubmissionOptions
- * @property {string} [date=0] - A timestamp in seconds or millseconds or in `MMM dd, yyyy h:mm:ss a` format. Defaults to `0` which will request all data
- * @property {string} [format='json'] - Format the submission data typee, It can be in `csv` or `json`. Defaults to `json` (JSON response)
- * @property {string} status - (Opt)Review status. Can be either, `approved`, `rejected`, `pending` or combine eg `approved|rejected`.
+ * Options provided to `fetchSubmissions()`
+ * @typedef {Object} FetchSubmissionOptions
+ * @property {string} [date=0] - Fetches only submissions from this timestamp.
+ * All values will be converted to surveyCTO `MMM dd, yyy h:mm:ss` format (in UTC time) in the request.
+ * Unix and Epoch timestamps are supported, as well as ISO date representatons.
+ * If set to 0, all submissions will be retrieved.
+ * @property {string} [format=json] - Format the submission data type as  `csv` or `json`.
+ * @property {string} [status] - Review status. Can be either, `approved`, `rejected`, `pending` or combine eg `approved|rejected`.
  */
 
 /**
  * Fetch form submissions
  * @example <caption>Fetch all form submissions</caption>
  * fetchSubmissions('test');
- * @example <caption> With `MMM dd, yyyy h:mm:ss a` date format</caption>
```

Comment:
These examples have been trimmed down so that each one only shows one thing. Ie, the CSV example doesn't include a date.